packages/shared/trade/src/transaction-sidebar/transaction-sidebar.tsx

```diff
@@ -104,10 +135,26 @@ export const TransactionSidebar = ({
     }
   };
 
-  return confirm ? (
-    <Confirmation2
+  const handleActionSidebarCancel = () => {
+    if (onCancelCallback) {
+      onCancelCallback();
+    }
```

Comment:
Should this be an else if?

Reply:
No, both of these need to have the possibility of running on one of the other